apollo-router/src/compute_job/mod.rs

```diff
@@ -1,11 +1,19 @@
+mod metrics;
+
 use std::future::Future;
 use std::ops::ControlFlow;
 use std::sync::OnceLock;
+use std::time::Instant;
 
 use opentelemetry::metrics::MeterProvider as _;
 use opentelemetry::metrics::ObservableGauge;
 use tokio::sync::oneshot;
 
+use self::metrics::ActiveComputeMetric;
+use self::metrics::JobWatcher;
+use self::metrics::Outcome;
+use self::metrics::observe_compute_duration;
+use self::metrics::observe_queue_wait_duration;
 use crate::ageing_priority_queue::AgeingPriorityQueue;
 pub(crate) use crate::ageing_priority_queue::Priority;
```

Comment:
```suggestion
use crate::ageing_priority_queue::Priority;
```
No longer used in public api of `compute_job` module